index.js

```diff
@@ -53,6 +53,8 @@ function registerListener(session, options, callback = () => {}) {
 
 		if (!options.saveAs) {
 			item.setSavePath(filePath);
+		} else if (options.saveAs) {
```

Comment:
The `if` here is moot.

Reply:
I know but see https://github.com/sindresorhus/electron-dl/pull/129/commits/08ae24c49500228e027cc3117f6fa7de7577c24f

Your formatting complains about negated conditions

Reply:
Yes. You fixed it incorrectly. Just invert the if.

Reply:
Didnt use the negated condition check before, now i know what it means. 
I just inverted the whole if to create the if/else.

The fix i did for the negated condition check was indeed incorrect. Fixed it now., sorry for the extra commit thought i could just edit it with the inline git editor, but created some extra spaces there :(